cv_next/lib/definitions.ts

```diff
@@ -93,3 +94,21 @@ export const Credits = [
   "Assaf Kabesa",
   "Alon Green",
 ];
+
+export const External_Credits = [
```

Comment:
Why do this manually and not, e.g. use the github api to extract the conttributors?

Reply:
Using actions or regular API from the typescript?

Besides, some contributors were only in gitlab and this part is mainly for the current and former "core" team and not randoms

Reply:
Oh I see, this isn't meant to be git-related contributors (like on the main page of the github project), but simply showing legacy contributors? Like, the specific names and contributions you put are not random but are meant to be hard-coded like the core team values?

Reply:
External are people that participated in the bug bounty and those that will help with PRs in the future as it's OS

Team are more core people or people that used to be really into the project

Reply:
I see.
Bug bounty - that surely has to be hard-coded. But "help with PRs" - that's very easily fetched via [api](https://docs.github.com/en/rest/collaborators).
If you don't want, that's absolutely fine, just thought it'd be a nice idea to prevent manual changes to things that can be automated :)

Reply:
I've added an issue for this to happen in a future version